macros/multiple_databases/datediff.sql

```diff
@@ -4,6 +4,8 @@
     datediff({{ datepart }}, {{ start_date_field }}, {{ end_date_field }})
 {%- elif target.type == 'sqlserver' -%}
     datediff_big({{ datepart }}, {{ start_date_field }}, {{ end_date_field }})
+{%- elif target.type == 'databricks' -%}
+    datediff({{ datepart }}, {{ start_date_field }}, {{ end_date_field }})
```

Comment:
`datediff` is a Databricks specific function and not a Spark SQL function according to the SQL guide.

When googling on Spark SQL datediff functions I come across some suggestions to cast both start and end to timestamps and compute the difference based on that.

Reply:
> When googling on Spark SQL datediff functions I come across some suggestions to cast both start and end to timestamps and compute the difference based on that.
Let's discuss this because I didn't find this myself and I tested with both dates and timestamp and it works 🤔.